src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -145,6 +145,14 @@ public static Configuration mapJdbcCatalogToConfig(Configuration ingestConfig, S
         if(tableList == null || tableList.isEmpty()){
             throw new RuntimeException(String.format("%s schema does not  have %s in source system",jdbc.getSchema(), jdbc.getTableTypes().toString()));
         }
+        if(jdbc.getIgnoreTables() != null && !jdbc.getIgnoreTables().isEmpty()){
+           tableList = tableList.stream().
+                   filter(table -> !jdbc.getIgnoreTables().contains(table.getName()))
+                   .collect(Collectors.toList());
+        }
+        if(tableList.isEmpty()){
+            throw new RuntimeException(String.format("No table to process. All tables are ignored. Ignored tables: %s", jdbc.getIgnoreTables()));
```

Comment:
Should be 

“no tables found or all tables ignored”?

Reply:
updated